eval_ckpts/NestingLayer.py

```diff
@@ -0,0 +1,77 @@
+import torch
```

Comment:
Can we rename this. If we use nesting in the title, or classes, it would be confusing, can we rename them to something else?